config/application.yml.example

```diff
@@ -26,5 +26,8 @@ MVI_PROCESSING_CODE: 'T'
 MOCK_MVI_SERVICE: 'true'
 GOV_DELIVERY_SERVER: stage-tms.govdelivery.com
 
+HEALTH_CARE_APPLICATION_CERTIFICATE_FILE: ~
```

Comment:
Other keys in this file are (mostly) oriented toward the remote service name. So "MHV_blah" instead of "PRESCRIPTIONS_blah". So I think we should have:
```
ES_URL
ES_CLIENT_CERT_PATH
ES_CLIENT_KEY_PATH
```
`ES` could be expanded to `ENROLLMENT_SERVICE` if you're feeling verbose.